app/CMakeLists.txt

```diff
@@ -238,13 +238,18 @@ if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_VecGeom)
     set(_driver "${CMAKE_CURRENT_SOURCE_DIR}/demo-loop/simple-driver.py")
     set(_gdml_inp "${PROJECT_SOURCE_DIR}/app/demo-loop/simple-cms.gdml")
     set(_hepmc3_inp "${PROJECT_SOURCE_DIR}/app/demo-loop/input.hepmc3")
+    if(CELERITAS_USE_Geant4)
+      set(_geant_exporter_exe "$<TARGET_FILE:geant-exporter>")
+    else()
+      set(_geant_exporter_exe "geant-exporter")
+    endif()
     add_test(NAME "app/demo-loop"
       COMMAND "$<TARGET_FILE:Python::Interpreter>"
       "${_driver}" "${_gdml_inp}" "${_hepmc3_inp}"
     )
     set(_env
       "CELERITAS_DEMO_EXE=$<TARGET_FILE:demo-loop>"
-      "CELERITAS_GEANT_EXPORTER_EXE=$<TARGET_FILE:geant-exporter>"
+      "CELERITAS_GEANT_EXPORTER_EXE=${_geant_exporter_exe}"
```

Comment:
I'd prefer the first since it has less duplication. If the nonexistent filename does get through, then we'll also get a sensible error from the downstream code like "failed to open geant-exporter: file does not exist" or whatever.

Reply:
That's the best case scenario.  In the worst case scenario it find an executable with that name on the PATH (which maybe from an other celeritas installation or from something completely unrelated (`geant-exporter` is somewhat of a generic name) (and then 'weird' things ensues)  .

Humm .. maybe a compromise is to use:

```
    if (TARGET geant-exporter)
      set(_geant_exporter_exe_def "CELERITAS_GEANT_EXPORTER_EXE=$<TARGET_FILE:geant-exporter>")
    else()
      set(_geant_exporter_exe_def "")
    endif()
```

Reply:
Fair enough! I'll add the change.